demisto_sdk/tests/integration_tests/validate_integration_test.py

```diff
@@ -5,50 +5,103 @@
 from demisto_sdk.commands.common.git_tools import git_path
 
 VALIDATE_CMD = "validate"
-AZURE_FEED_PACK_PATH = join(git_path(), "demisto_sdk/tests/test_files/content_repo_example/Packs/FeedAzure")
+TEST_FILES_PATH = join(git_path(), "demisto_sdk/tests/test_files")
+AZURE_FEED_PACK_PATH = join(TEST_FILES_PATH, "content_repo_example/Packs/FeedAzure")
 
 
-def test_integration_validate_incident_field_positive():
+def assert_positive(file_path, result):
     """
-    Given
-    - Valid `city` incident field.
-
-    When
-    - Running validation on it.
-
-    Then
-    - Ensure validation passes.
-    - Ensure success validation message is printed.
+    Series of asserts every positive test should do
+    :param file_path: path to the file
+    :param result: result object as returned from runner.invoke
     """
-    pack_incident_field_path = join(AZURE_FEED_PACK_PATH, "IncidentFields/incidentfield-city.json")
-    runner = CliRunner(mix_stderr=False)
-    result = runner.invoke(main, [VALIDATE_CMD, "-p", pack_incident_field_path])
     assert result.exit_code == 0
     assert "Starting validating files structure" in result.stdout
-    assert f"Validating {pack_incident_field_path}" in result.stdout
+    assert f"Validating {file_path}" in result.stdout
     assert "The files are valid" in result.stdout
     assert result.stderr == ""
 
 
-def test_integration_validate_integration_negative():
-    """
-    Given
-    - FeedAzure integration with non-latest docker image.
+class TestIncidentField:
+    def test_positive(self):
+        """
+        Given
+        - Valid `city` incident field.
 
-    When
-    - Running validation on it.
+        When
+        - Running validation on it.
 
-    Then
-    - Ensure validation fails.
-    - Ensure failure message on non-latest docker image.
-    """
-    pack_integration_path = join(AZURE_FEED_PACK_PATH, "Integrations/FeedAzure/FeedAzure.yml")
-    runner = CliRunner(mix_stderr=False)
-    result = runner.invoke(main, [VALIDATE_CMD, "-p", pack_integration_path])
-    assert result.exit_code == 1
-    assert "Starting validating files structure" in result.stdout
-    assert f"Validating {pack_integration_path}" in result.stdout
-    assert "The docker image tag is not the latest, please update it" in result.stdout
-    assert f"{pack_integration_path}: You're not using latest docker for the file, " \
-           "please update to latest version." in result.stdout
-    assert result.stderr == ""
+        Then
+        - Ensure validation passes.
+        - Ensure success validation message is printed.
+        """
+        pack_incident_field_path = join(AZURE_FEED_PACK_PATH, "IncidentFields/incidentfield-city.json")
+        runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(main, [VALIDATE_CMD, "-p", pack_incident_field_path])
+        assert_positive(pack_incident_field_path, result)
+
+
+class TestIntegration:
+    @staticmethod
```

Comment:
no need for the staticmethod decorators as these methods arent really called from anywhere

Reply:
true, it was mainly to remove the need for self. I'll remove those